covid_graphs/covid_graphs/country_graph.py

```diff
@@ -62,15 +62,17 @@ def __init__(
             self.t = np.arange(len(self.cumulative_active)) + 1
 
     def create_country_figure(self):
+        colors = ['SteelBlue', 'Purple', 'Green'][:len(self.curves)]
+
         shapes = [
             # Add vertical dotted lines marking the maxima
             dict(type="line",
-                 yref="paper",
                  x0=curve.t[curve.maximal_idx],
                  y0=0,
```

Comment:
Nemali by sme dat y0>0? Mam pocit ze semilog a loglog grafy vyzeraju divne... ale neviem ci to je tymto.

Reply:
Yes, but the `yaxis` range should be also reduced to `[1, max_value]`. Without it it looks even weirder.